.ci/scripts/gen_release_jobs.py

```diff
@@ -24,7 +24,7 @@
 
 CHANNELS = args.channels
 PYTHON_VERSIONS = ["3.9", "3.10", "3.11"]
-SYSTEMS = ["ubuntu-latest", "windows-latest"]
+SYSTEMS = ["ubuntu-22.04", "windows-2022"]
```

Comment:
I think it worth to add some comment for better understanding and justification why not the latest versions is used.

Reply:
Added.